components/Header.tsx

```diff
@@ -1,74 +1,130 @@
 // Copyright 2023 the Deno authors. All rights reserved. MIT license.
 import {
   ACTIVE_LINK_STYLES,
-  BUTTON_STYLES,
   LINK_STYLES,
-  NAV_STYLES,
   SITE_BAR_STYLES,
   SITE_NAME,
 } from "@/utils/constants.ts";
 import { stripe } from "@/utils/payments.ts";
-import { Bell, CircleFilled } from "./Icons.tsx";
-import { getToggledStyles } from "@/utils/display.ts";
+import { Bars, Bell, CircleFilled, Cross } from "./Icons.tsx";
+import { cx } from "@twind/core";
```

Comment:
Sidenote: this might be able to replace the new `getToggledStyles()`.

Reply:
Yes, we can use it. I was just writing code like this at first:

```jsx
class={cx(
  getToggledStyles(
    LINK_STYLES,
    ACTIVE_LINK_STYLES,
    props.url.pathname === "/pricing",
  ),
  NAV_ITEM,
)}
```

I noticed that this code can be rewritten as: 

```jsx
class={cx(
  props.url.pathname === "/pricing" ? ACTIVE_LINK_STYLES : LINK_STYLES
  NAV_ITEM,
)}
```

This cx style description is used in deno deploy and can handle both concatenation and toggle of CSS strings. 
Since this header component uses cx throughout, I removed getToggledStyles as a dependency.

I think it would be better to write getToggledStyles with a ternary operator where cx is used, what do you think?

Reply:
What about if we got rid of `getToggledStyles()` completely across the codebase and used `cx()` instead?

Reply:
yeah if it's okay, I'll like to help to do this